pkg/cluster/describe_test.go

```diff
@@ -0,0 +1,40 @@
+package cluster
+
+import (
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"testing"
+)
+
+// newTestCluster assembles a *cmv1.Cluster while handling the error to help out with inline test-case generation
+func newTestCluster(t *testing.T, cb *cmv1.ClusterBuilder) *cmv1.Cluster {
+	cluster, err := cb.Build()
+	if err != nil {
+		t.Fatalf("failed to build cluster: %s", err)
+	}
+
+	return cluster
+}
+
+func TestFindHyperShiftMgmtSvcClusters(t *testing.T) {
+	tests := []struct {
+		name         string
+		cluster      *cmv1.Cluster
+		expectedMgmt string
+		expectedSvc  string
+	}{
+		{
+			name:    "Not HyperShift",
+			cluster: newTestCluster(t, cmv1.NewCluster().Hypershift(cmv1.NewHypershift().Enabled(false))),
+		},
```

Comment:
The current test-case covers the really simple case where this is not a HyperShift cluster and I'm not sure how to mock `conn *sdk.Connection` so that I can unit test HyperShift clusters. If there are any tips or examples I am happy to do so